.circleci/config.yml

```diff
@@ -111,6 +111,9 @@ jobs:
           cache-version: v1
       - revenuecat/trust-github-key
       - revenuecat/setup-git-credentials
+      - run:
+          name: Switching to main branch
+          command: git checkout main
```

Comment:
Hmm I see a problem with doing it from `main`... Imagine that we tag a major version `6.0.0`. On `main` we would still be in `5.x.x`, so this would bump to a `5.x+1.x-SNAPSHOT`. Ideally we would run this job whenever we merge a release branch into main, but I don't think we can do that trigger in circleci right?

Reply:
Yes, you are correct. Ideally we would do that yes, but I don't know how... I guess we could run it on every commit to main? and if the version is not a snapshot version trigger the job? What do you think?

Reply:
Hmm that could work yeah... I like it!